### ISSUE TITLE
Section "How to get ATP anti-phishing" contains reference to ATP Safe Links

### ISSUE THREAD

**Issue by willseward** [MicrosoftDocs/OfficeDocs-o365seccomp#328]
The aforementioned section is copy-and-pasted from another page, but the content was not changed to reflect it's new home.

https://docs.microsoft.com/en-us/office365/securitycompliance/atp-safe-links

&gt;ATP Safe Links features are part of Advanced Threat Protection, which...

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 89997683-62da-9fc5-068b-b5a48b239c16
* Version Independent ID: b80d0844-3f5a-5e19-f1c8-9badee2132ba
* Content: [ATP anti-phishing capabilities in Office 365](https://docs.microsoft.com/en-us/office365/securitycompliance/atp-anti-phishing)
* Content Source: [SecurityCompliance/atp-anti-phishing.md](https://github.com/microsoftdocs/officedocs-o365seccomp/blob/public/SecurityCompliance/atp-anti-phishing.md)
* Service: **o365-administration**
* GitHub Login: @KCCross
* Microsoft Alias: **krowley**

**Comment by denisebmsft** [comment]
@willseward you are correct. Thank you so much for catching that. I have it in the queue to fix.
